lib/OrdinaryDiffEqStabilizedRK/src/algorithms.jl

```diff
@@ -62,66 +67,74 @@ for Alg in [:ESERK4, :ESERK5, :RKC]
+    "ESERK4",
+    "Stabilized Explicit Method.",
+    """J. Martín-Vaquero, B. Kleefeld. Extrapolated stabilized explicit Runge-Kutta methods,
+    Journal of Computational Physics, 326, pp 141-155, 2016. doi:
+    https://doi.org/10.1016/j.jcp.2016.08.042.""",
+    """
+    - `eigen_est`: function of the form
+        `(integrator) -> integrator.eigen_est = upper_bound`,
+        where `upper_bound` is an estimated upper bound on the spectral radius of the Jacobian matrix.
+        If `eigen_est` is not provided, `upper_bound` will be estimated using the power iteration.
+    """,
+    """
+    eigen_est = nothing,
+    """)
 function ESERK4 end
 
-"""
-    ESERK5(; eigen_est = nothing)
-
-J. Martín-Vaquero, A. Kleefeld. ESERK5: A fifth-order extrapolated stabilized explicit Runge-Kutta method,
-Journal of Computational and Applied Mathematics, 356, pp 22-36, 2019. doi:
-https://doi.org/10.1016/j.cam.2019.01.040.
-
-ESERK5: Stabilized Explicit Method.
-Fifth order extrapolated stabilized Runge-Kutta method.
-Exhibits high stability for real eigenvalues and is smoothened to allow for moderate sized complex eigenvalues.
-
-This method takes the keyword argument `eigen_est` of the form
-
-`eigen_est = (integrator) -> integrator.eigen_est = upper_bound`,
-
-where `upper_bound` is an estimated upper bound on the spectral radius of the Jacobian matrix.
-If `eigen_est` is not provided, `upper_bound` will be estimated using the power iteration.
-"""
+@doc generic_solver_docstring("""Fifth order method. Exhibits high stability for real eigenvalues
+    and is smoothened to allow for moderate sized complex eigenvalues.""",
+    "ESERK5",
+    "Stabilized Explicit Method.",
+    """J. Martín-Vaquero, A. Kleefeld. ESERK5: A fifth-order extrapolated stabilized explicit Runge-Kutta method,
+    Journal of Computational and Applied Mathematics, 356, pp 22-36, 2019. doi:
+    https://doi.org/10.1016/j.cam.2019.01.040.""",
+    """
+    - `eigen_est`: function of the form
+        `(integrator) -> integrator.eigen_est = upper_bound`,
+        where `upper_bound` is an estimated upper bound on the spectral radius of the Jacobian matrix.
+        If `eigen_est` is not provided, `upper_bound` will be estimated using the power iteration.
+    """,
+    """
+    eigen_est = nothing,
+    """)
 function ESERK5 end
 
+@doc generic_solver_docstring("""Second order method. description TBD""",
+    "SERK2",
+    "Stabilized Explicit Method.",
+    """REF TBD""",
+    """
+    - `controller`: TBD
+    - `eigen_est`: function of the form
+        `(integrator) -> integrator.eigen_est = upper_bound`,
+        where `upper_bound` is an estimated upper bound on the spectral radius of the Jacobian matrix.
+        If `eigen_est` is not provided, `upper_bound` will be estimated using the power iteration.
+    """,
+    """
+    controller = :PI
+    eigen_est = nothing,
+    """)
 struct SERK2{E} <: OrdinaryDiffEqAdaptiveAlgorithm
```

Comment:
@ChrisRackauckas This docstring is new, and still has a blank description and reference.

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/pull/732 https://onlinelibrary.wiley.com/doi/10.1002/num.21704

The papers show that it should be close to ROCK2 but generally we've seen the ROCK integrators perform a bit better.